packages/@lightningjs/ui-components/src/components/ScrollWrapper/ScrollSlider.js

```diff
@@ -27,4 +27,8 @@ export default class ScrollSlider extends Slider {
   static get __themeStyle() {
     return styles;
   }
+
+  _init() {
+    this.announce = '';
```

Comment:
I'm trying to decide if I feel better about this here, or in the ScrollWrapper. It's still overridable since it's in _init, but I'm on the fence. I know this isn't used anywhere else, but just in case

Reply:
I understand that, I'm just thinking about explicitly setting the default here in `_init`. I also feel like we usually try to set defaults in `_construct`, but I'll defer to @erautenberg on that

Reply:
Just caught up. I think moving this from init to construct makes sense, and maybe we can do `this._announce = '';` instead.